Assets/SteamVR_Unity_Toolkit/Examples/Resources/Scripts/RendererOffOnDash.cs

```diff
@@ -0,0 +1,52 @@
+namespace VRTK
```

Comment:
Take the example script out of the VRTK namespace, it should just include the VRTK namespace with:
`using VRTK;`